src/commands/sensor/models/bodies/location-body.ts

```diff
@@ -1,39 +1,38 @@
+import { Type } from 'class-transformer';
 import { ApiProperty } from '@nestjs/swagger';
-import { IsString, IsNumber, IsNotEmpty, IsOptional } from 'class-validator';
+import { IsString, IsNumber, IsNotEmpty, IsOptional, Min, Max } from 'class-validator';
 
 export class LocationBody {
```

Comment:
In de query is `baseObjectId` verplaatst uit het 'location' object. Zou dat hier ook moeten / kunnen / beter zijn?

Reply:
Het valt nog wel onder de locationbody, je wilt m ook kunnen updaten als je relocated uitvoert. Omdat je in mongodb direct een stukje geojson opslaat als location en daar de index op gebouwt wordt moest baseObject er uit. Lijkt mij makkelijk om het zo te laten, anders moet je twee locationbodies hebben, een voor registerSensor en een voor relocate.